utils/watcher_strace.py

```diff
@@ -107,6 +113,20 @@
                                         item)
                                     break
                 # For file access, put it here to ignore sendmsg
+                # For music file being set to play
+                elif 'setMedia()' in output:
```

Comment:
I was thinking to separate supressor from other codes, so if we need more write events, we can modify the supressor easily.